.github/workflows/ci.yml

```diff
@@ -251,16 +251,3 @@ jobs:
           name: "test-results"
           path: |
             results/Documentation/renderer/*
-
-  devcontainer-check:
```

Comment:
Can you please explain why you've been remove this check?

Reply:
Because I created new workflow and it would be duplicated if I remain it.